tests/test_radiation.py

```diff
@@ -0,0 +1,173 @@
+"""Test module for abiotic.radiation.py."""
+
+from contextlib import nullcontext as does_not_raise
+
+import numpy as np
+import pytest
+
+from virtual_rainforest.core.model import InitialisationError
+from virtual_rainforest.models.abiotic import radiation
+
+
+@pytest.mark.parametrize(
+    argnames="elev, exp_err, exp_message",
+    argvalues=[
+        pytest.param(np.array([100.0, 1000.0]), does_not_raise(), ()),
+        pytest.param(
+            np.array([-100.0, 1000.0]),
+            pytest.raises(InitialisationError),
+            ("Initial elevation contains at least one negative value!"),
+        ),
+    ],
+)
+def test_radiation_class(elev, exp_err, exp_message):
+    """Test elevation above zero."""
+
+    with exp_err:
+        radiation.data["elevation"] = elev
+        result = radiation.Radiation()
+
+        assert result.elevation.all() == elev.all()
```

Comment:
Failed: DID NOT RAISE <class 'virtual_rainforest.core.model.InitialisationError'>

Reply:
So you've told `pytest` that this block should raise an `InitialisationError` (for some parameters) and it's complaining that it wasn't (either no error was raised or a different type was).

I have a suspicion as to why this is, but have you checked that the error is raised by doing something like this outside of `pytest`?